src/components/Helpers/ButtonSolid/ButtonSolid.tsx

```diff
@@ -3,11 +3,19 @@ import css from "./ButtonSolid.module.scss";
 
 interface ButtonSolidProps {
   color?: string;
+  isIcon?: boolean;
+  isDisabled?: boolean;
   children: ReactElement;
+  onClick: any;
 }
 
-export const ButtonSolid = ({ color = "", children }: ButtonSolidProps) => {
-  const modifier = css[color] ? css[color] : "";
+export const ButtonSolid = ({ color = "", isIcon, isDisabled, onClick, children }: ButtonSolidProps) => {
```

Comment:
I'm not sure about the approach taken with color and isIcon property. I will focus on isIcon as it has been added.

I would suggest to make a new component for Icon (ButtonIcon ?) a la emotion:

```
export const ButtonIcon = styled(Button)` ... `;
```

(I'm not saying to use emotion, use what you want, just pointing how I think we should do)

Reply:
ok got it, updated